db/schema.rb

```diff
@@ -192,7 +192,7 @@
     t.text "strengths"
     t.text "skills"
     t.text "interests"
-    t.string "state", default: "interested/registered"
+    t.string "state", default: "registered"
```

Comment:
I've just seen what you mean, because you've also set a different default `interested` in your branch. But I adopted the same state collection as used in the clients model as well, and there it is registered, not interested.